flattentool/input.py

```diff
@@ -200,7 +204,9 @@ def merge(base, mergee, debug_info=None):
                             + id_info
                         )
                     warn(
-                        'You may have a duplicate Identifier: We couldn\'t merge these rows with the {}: field "{}" in sheet "{}": one cell has the value: "{}", the other cell has the value: "{}"'.format(
+                        _(
+                            'You may have a duplicate Identifier: We couldn\'t merge these rows with the {}: field "{}" in sheet "{}": one cell has the value: "{}", the other cell has the value: "{}"'
+                        ).format(
                             id_info,
                             key,
                             debug_info.get("sheet_name"),
```

Comment:
I know this is quite a pain to change, but wrg to recent discussion having more context for translators might be good especially with multiple format strings. Perhaps where it is really non obvious from the en context use the %(blah)s method?

Reply:
We discussed this, and will open a new issue for this.